pkgs/desktops/lxqt/lxqt-panel/default.nix

```diff
@@ -79,6 +80,13 @@ stdenv.mkDerivation rec {
     solid
   ];
 
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/lxqt/lxqt-panel/pull/2166.patch";
```

Comment:
Upstream made a point release  include this patch
https://github.com/lxqt/lxqt-panel/releases/tag/2.1.1

Reply:
Updated.